### ISSUE TITLE
Shell completion --profile using named profiles in ~/.aws/config 

### ISSUE THREAD

**Issue by cbcoutinho** [aws/aws-cli#5580]
**Is your feature request related to a problem? Please describe.**
I have to type in the profile name manually whereas all the other options for the cli are already available

**Describe the solution you'd like**

```shell
$ aws --profile <TAB>
<profile-name-1>
<profile-name-2>
<profile-name-3>
```

**Describe alternatives you've considered**
There must be a way to simply grep the config file to get a list of available profiles, like this, but then using the awscli.completer library.

```shell
$ grep -oP 'profile \K[\w-]+' ~/.aws/config
```

**Additional context**

**Comment by kdaily** [comment]
Thanks for the feature request, @cbcoutinho!

**Comment by cbcoutinho** [comment]
This is partially implemented by the `--cli-auto-prompt` #5664

**Comment by vz10** [comment]
HI @cbcoutinho , 

The AutoPrompt feature has been released in AWS CLI version 2.1.0 this release includes completion for `--profile` (without autoprompt)  as well. 

Please try it out and let us know if it's fixed your issue.

Thanks

**Comment by cbcoutinho** [comment]
Hi @vz10 - indeed it does, thanks for the speedy response to this request!
